src/Url.js

```diff
@@ -52,7 +52,7 @@ const Url = React.createClass({
 
   onClick(event) {
     event.preventDefault();
-    ActionCreator.navigateTo(this.props);
+    Store.dispatch(ActionCreator.navigateTo(this.props));
```

Comment:
Have been thinking about the Smart Components/Dumb Components/react-redux thing...Atm it seems to me that ComponentRouters/Locations/URLs all have to be "Smart" and talk to the Store as they can all be Top Level Components (as in the example App). I cant think of a way to get around that without changing the current API - such as wrapping everything in another top level component.

React-Redux it seems we could use in either case - in the first case though we would essentially just be wrapping all the existing Stuff with Smart Containers. And users would then have to wrap their whole App with the Provider Component so its not a change that would be hidden from current users anyway.

So, unless I'm missing something (very much possible), I agree that it doesnt seem worth it right now unless we ensure that everything lives inside a new Top Level Container.

Anyway, will add more tests to Store and push.